syncopy/tests/synth_data.py

```diff
@@ -29,26 +30,47 @@ def collect_trials(trial_generator):
     If the underlying trial generating function also accepts
     a `samplerate`, forward this directly.
 
+    If the underlying trial generating function also accepts
+    a `seed`, forward this directly. One can pass a single seed,
+    to be used for all trials, or a list/np.ndarray of seeds with
+    len/size equal to nTrials, with one seed per trial. Leaving
+    at the default value, None, will select a random seed each time,
+    (and it will differ between trials).
+
     The default `nTrials=None` is the identity wrapper and
     just returns the output of the trial generating function
     directly, so a single trial :class:`numpy.ndarray`.
     """
 
     @functools.wraps(trial_generator)
-    def wrapper_synth(nTrials=None, samplerate=1000, **tg_kwargs):
+    def wrapper_synth(nTrials=None, samplerate=1000, seed=None, seed_per_trial=True, **tg_kwargs):
```

Comment:
What about setting a default parameter for `seed`, such that we automatically inject a stable seed into all our current and future tests?

Reply:
done